src/lib.rs

```diff
@@ -64,20 +64,63 @@ pub mod raftstore;
 pub mod server;
 pub mod storage;
 
+// Detect tikv enabled features.
+fn detect_enabled_features() -> String {
+    macro_rules! detect {
+        ($f:expr, $feat:tt) => {
+            if cfg!(feature = $feat) {
+                $f.push_str($feat);
+                $f.push(' ');
+            }
+        };
+    }
+
+    let mut flags = String::new();
```

Comment:
Rather than hardcode this, can you use one of the toml crates to parse Cargo.toml for the features section instead?

Reply:
I think the Cargo.toml is not available at run time.

Reply:
Maybe setting it to 'Unknown' when Cargo.toml is not available is a better solution?